client/state/ui/editor/selectors.js

```diff
@@ -27,3 +38,35 @@ export function isEditorNewPost( state ) {
 export function isEditorDraftsVisible( state ) {
 	return state.ui.editor.showDrafts;
 }
+
+/**
+ * Returns the editor URL path for the given site ID, post ID pair.
+ *
+ * @param  {Object} state  Global state tree
+ * @param  {Number} siteId Site ID
+ * @param  {Number} postId Post ID
+ * @return {String}        Editor URL path
+ */
+export function getEditorPath( state, siteId, postId ) {
+	const type = get( getSitePost( state, siteId, postId ), 'type', 'post' );
+
+	let path;
+	switch ( type ) {
+		case 'post': path = '/post'; break;
+		case 'page': path = '/page'; break;
+		default: path = `/type/${ type }`; break;
```

Comment:
>I agree `/view` could be a bit ambiguous - I think `/list` is good though.

I'll plan to create a follow-up pull request specifically for starting a dialogue around these paths, as I'd prefer to avoid it holding up this one.